core/modules/entity/entity.controller.inc

```diff
@@ -27,6 +27,15 @@ interface EntityControllerInterface {
    */
   public function resetCache(array $ids = NULL);
 
+  /**
+   * Resets the internal, static entity cache.
+   *
+   * @param string[] $ids
```

Comment:
Hm, yeah we do this when returning an array of objects sometimes, e.g. `Node[]` or `File[]` indicates an array of nodes or files. I haven't seen this for basic types like strings though. Still it seems as though it may be more informative than just saying `array`.

Reply:
For now I just set this to `array` to match the rest of core.